go.mod

```diff
@@ -4,10 +4,12 @@ go 1.19
 
 require (
 	github.com/Masterminds/semver/v3 v3.2.0
+	github.com/cert-manager/cert-manager v1.11.0
 	github.com/gardener/component-spec/bindings-go v0.0.78
 	github.com/go-logr/logr v1.2.3
 	github.com/go-logr/zapr v1.2.3
 	github.com/kyma-project/module-manager v0.0.0-20230203122329-275f0c18b284
+	github.com/kyma-project/module-manager/operator v0.0.0-20221125150159-5e84c4855249
```

Comment:
why is this import back? :O this should not be here, maybe a malformed import?

Reply:
Removed